src/resources/Groups/GroupsInterfaces.ts

```diff
@@ -0,0 +1,93 @@
+export interface GroupModel {
+    id: string;
+    displayName: string;
+    invites?: InviteModel[];
+    members?: MemberModel[];
+    privileges?: PrivilegeModel[];
+    realms?: RealmModel[];
+    deletable?: boolean;
+    resourceId?: string;
+    groupsThatCanEdit?: IdAndDisplayNameModel[];
+    apiKeysThatCanEdit?: IdAndDisplayNameModel[];
+}
+
+export interface IdAndDisplayNameModel {
```

Comment:
This is generic and you will find it elsewhere in the api, maybe here is not the place

Reply:
Noted. I'll let it there for now, but when another resource will require this type we'll move it